community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/slurmsync.py

```diff
@@ -534,6 +534,60 @@ def sync_maintenance_reservation(lkp: util.Lookup) -> None:
       create_reservation(lkp, res_name, node, start_time)
 
 
+def delete_maintenance_job(job_name: str) -> None:
+    util.run(f"scancel --name={job_name}")
+
+
+def create_maintenance_job(job_name: str, node: str) -> None:
+    util.run(f"sbatch --job-name={job_name} --nodelist={node} /slurm/custom_scripts/perform_maintenance.sh")
+
+
+def get_slurm_maintenance_job(lkp: util.Lookup) -> Dict[str, str]:
+    res = util.run(f"{lkp.scontrol} show jobs --json")
```

Comment:
* There is already another place in code that fetches all jobs, unify and add cache to it;
* Please use `dataclass Job` to represent jobs. 
* `scontrol show nodes --json` is ~100 times slower than `scontrol show nodes`, please test if same happens with `scontrol show jobs`;

Reply:
yes, discussed offline. This behavior is same across other APIs as well. Serializing using json is much slower.

Reply:
Refactored it.